understanding/non-text-content.html

```diff
@@ -183,82 +170,57 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
                   
-                  <dt>Goal</dt>
-                  <dd>Non-text information is available to more people.</dd>
+                  <dt>目標</dt>
+                  <dd>非テキスト情報をより多くの人々が利用できるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Create a text alternative for visual and auditory content.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>視覚的および聴覚的なコンテンツに対してテキストによる代替を作成する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who can’t fully see or hear content can understand it.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>コンテンツを見たり聞いたりすることが完全にはできない人々が、内容を理解できるようにするため。</dd> 
```

Comment:
「ようにするため」は原文では言っておらず、少し補いすぎのように感じました。素直にやるとこうでしょうか。

```suggestion
                  <dd>コンテンツを見たり聞いたりすることが完全にはできない人々が、内容を理解できる。</dd> 
```